[2023-05-29 13:24:35,288] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.task2 manual__2023-05-29T13:24:31.188023+00:00 [queued]>
[2023-05-29 13:24:35,304] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: my_dag.task2 manual__2023-05-29T13:24:31.188023+00:00 [queued]>
[2023-05-29 13:24:35,304] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-29 13:24:35,305] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-05-29 13:24:35,305] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-29 13:24:35,335] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): task2> on 2023-05-29 13:24:31.188023+00:00
[2023-05-29 13:24:35,340] {standard_task_runner.py:52} INFO - Started process 259 to run task
[2023-05-29 13:24:35,344] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task2', 'manual__2023-05-29T13:24:31.188023+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp4eyeg26i', '--error-file', '/tmp/tmpwguq18e3']
[2023-05-29 13:24:35,345] {standard_task_runner.py:80} INFO - Job 3: Subtask task2
[2023-05-29 13:24:35,419] {task_command.py:369} INFO - Running <TaskInstance: my_dag.task2 manual__2023-05-29T13:24:31.188023+00:00 [running]> on host 579441c7a70b
[2023-05-29 13:24:35,547] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=task2
AIRFLOW_CTX_EXECUTION_DATE=2023-05-29T13:24:31.188023+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-29T13:24:31.188023+00:00
[2023-05-29 13:24:35,548] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-05-29 13:24:35,549] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "Task 2"']
[2023-05-29 13:24:35,560] {subprocess.py:85} INFO - Output:
[2023-05-29 13:24:35,562] {subprocess.py:92} INFO - Task 2
[2023-05-29 13:24:35,563] {subprocess.py:96} INFO - Command exited with return code 0
[2023-05-29 13:24:35,609] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task2, execution_date=20230529T132431, start_date=20230529T132435, end_date=20230529T132435
[2023-05-29 13:24:35,637] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-29 13:24:35,685] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
